### PR TITLE
Add missing AppContext Switches in WCF

### PR DESCRIPTION
**Pull request by jiayi11**
@Lxiamail

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 1]
```diff
@@ -99,6 +100,9 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
 |`Switch.System.ServiceModel.`<br/>`UseSha1InPipeConnectionGetHashAlgorithm`|Controls whether WCF uses a SHA1 or a SHA256 hash to generate random names for named pipes.|.NET Framework 4.7.1|
+|`Switch.System.ServiceModel.`<br/>`DisableExplicitConnectionCloseHeader`|When using the HTTP transport with a self hosted service, settings this value to true causes WCF to ignore an application adding the `Connection: close` header to the response headers for a request. Setting this value to false enables adding the `Connection: close` header to the response headers which results in the closing of the request socket after a response has been sent.|.NET Framework 4.6|
+|`Switch.System.ServiceModel.`<br/>`DontEnableSystemDefaultTlsVersions`|A value of false sets the default configuration to allow the operating system to choose the protocol. A value of true sets the default to the highest protocol available.|.NET Framework 4.7|
+|`Switch.System.IdentityModel.Internals`<br/>`IncludeNullExceptionMessageInETWTrace`|Controls whether to throw a NullReferenceException when the exception message is null.|.NET Framework 4.7|
```
Comment (Lxiamail) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: Should it be Switch.System.ServiceModel.Internals.IncludeNullExceptionMessageInETWTrace instead?

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 2]
```diff
@@ -99,6 +100,9 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
 |`Switch.System.ServiceModel.`<br/>`DisableUsingServicePointManagerSecurityProtocols`|Along with `Switch.System.Net.DontEnableSchUseStrongCrypto`, determines whether WCF message security uses TLS 1.1 and TLS 1.2.|.NET Framework 4.7 |    
 |`Switch.System.ServiceModel.`<br/>`UseSha1InMsmqEncryptionAlgorithm`|Determines whether the default message signing algorithm for MSMQ messages in WCF is SHA1 or SHA256.|.NET Framework 4.7.1|
 |`Switch.System.ServiceModel.`<br/>`UseSha1InPipeConnectionGetHashAlgorithm`|Controls whether WCF uses a SHA1 or a SHA256 hash to generate random names for named pipes.|.NET Framework 4.7.1|
+|`Switch.System.ServiceModel.`<br/>`DisableExplicitConnectionCloseHeader`|When using the HTTP transport with a self hosted service, settings this value to true causes WCF to ignore an application adding the `Connection: close` header to the response headers for a request. Setting this value to false enables adding the `Connection: close` header to the response headers which results in the closing of the request socket after a response has been sent.|.NET Framework 4.6|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: - This isn't in alphabetical order.
- self hosted --> self-hosted
- settings --> setting 
- true --> \`true\`
- false --> \`false\`
- headers which results in the closing of --> headers, which results in closing

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 3]
```diff
@@ -99,6 +100,9 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
+|`Switch.System.ServiceModel.`<br/>`DontEnableSystemDefaultTlsVersions`|A value of false sets the default configuration to allow the operating system to choose the protocol. A value of true sets the default to the highest protocol available. (Also available on servicing branch of previous framework versions)|.NET Framework 4.7.1|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: - This also should be in alphabetical order.
- false --> \`false\`
- true --> \`true\`

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 4]
```diff
@@ -99,6 +100,9 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
+|`Switch.System.ServiceModel.Internals`<br/>`IncludeNullExceptionMessageInETWTrace`|Controls whether to throw a NullReferenceException when the exception message is null.|.NET Framework 4.7|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: NullReferenceException should be a link: \<xref:System.NullReferenceException>

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for updating the list of supported switches, @yujayee. I've left a number of comments and suggested changes.

**Comment by yujayee** [comment]
@rpetrusha Thank you so much for your suggestions. I've fixed those errors according to your comments.

**Review by rpetrusha — APPROVED** [review]
Thanks, @yujayee, for making the additional changes. I'll merge your PR now.

**Pull request merged**